gensrc/thrift/PaloInternalService.thrift

```diff
@@ -576,6 +576,15 @@ struct TPipelineInstanceParams {
   6: optional i32 backend_num
 }
 
+struct TPipelineResourceGroup {
+  1: optional i64 id
+  2: optional string name
+  3: optional map<string, string> properties
+  4: optional i64 version
+  5: optional string parent_name
+  6: optional set<string> children_names
```

Comment:
1 Id should be used here;
2 Usually BE need not to know the child group.

Reply:
done